_posts/2017-08-08-Buffer_Overflows.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "Buffer Overflows from scratch"
```

Comment:
I will surely fix all those linguistic error 👍 
I wanted to extend the article , but then it requires good enough use of GDB so I preferred not to.
If you say , I will extend the post to a point of redirecting the flow of control to a completely "uncalled" function, for now it just demonstrates compromising a value stored in some variable.